### DISCUSSION TITLE
Can pygwalker handle multiple dataframes

### DISCUSSION THREAD

**Discussion by moltra** [Kanaries/pygwalker#276] [Q&A]
Can pygwalker handle mulitple dataframes?  I am wanting to limport 4 different dataframes (tables) and look at the columns and data to see what the relationships are between the tables.

**Comment by longxiaofei** [comment]
Hi, can you provide us with the specific operations you want to do on the UI?

Do you want to do some join operations on the UI to combine multiple tables into one table?

or something else?

**Reply by moltra** [reply]
I am trying to find a way to determine relationships between the different tables in the dataframes.  I am also looking for a way to get a good exploratory data analysis before, I start creating Power bi reports

**Comment by ObservedObserver** [comment]
It sounds like you can explore the dataframes separately in four pygwalker cells, or you want something else？

**Reply by moltra** [reply]
I am wanting to explore them together, so I can find the relationship and correlation columns.
